CONTRIBUTING.md

```diff
@@ -55,17 +55,16 @@ encouraged to try other DVC repositories -- especially real-world cases!
 
 - Go to the `demo/` directory of this repo.
 
-- It is recommended that you create the project's virtual environment with
-  Python 3.8 as some dependencies are not yet available for Python 3.9
+- Install [Yarn](https://yarnpkg.com/) if needed, and run `yarn install` to
+  setting up the project.
 
-- `yarn install` is a prerequisite for automatically setting up the demo
-  project.
+- Go back to the root of this repo (not `./demo`) and `yarn setup:venv` to setup
+  the Python virtual environment in the demo project.
 
-- Run `yarn setup:venv` from the monorepo root (not `./demo`) to automatically
-  setup the Python virtual environment in the demo project.
+  > Python 3.8 is recommended, as some dependencies are not yet available for
```

Comment:
hmm. I'm using Python 3.9 I think

Reply:
Same and so far no issues. This info was there already so I'm assuming it's correct.

Reply:
got it , I think we can remove it

Reply:
Gone!